src/pages/Dashboard.tsx

```diff
@@ -6,33 +6,71 @@ import { Toggle, PageLayout } from "../components";
 import {
   DashboardOverallStats,
   DashboardTable,
-  DashboardUserStats
+  DashboardUserStats,
+  DashboardFilterGroup
 } from "../components/Dashboard";
 import { useWalletModal } from "../providers/WalletModal";
-import { SignedMeetingsResponse } from "../types/meets";
+import { Meet, MeetFilters, SignedMeetingsResponse } from "../types/meets";
 import utils from "../utils";
 
+const AUS_NZ_LOCATIONS = [
```

Comment:
maybe this could be in the config endpoint in the BE?

Reply:
Not a bad idea, however these locations wont change (like the tracks do) and the BE doesn't require AUS_NZ_LOCATIONS for any of it's logic, so I think it is fine here.